submissions/catalytic_activity_concentrations.ttl

```diff
@@ -0,0 +1,113 @@
+  qudt:informativeReference "https://doi.org/10.1351/goldbook.C00882"^^xsd:anyURI ;
+  qudt:applicableUnit unit:KAT-PER-L ;
+  qudt:applicableUnit unit:MicroKAT-PER-L ;
+  qudt:applicableUnit unit:NanoKAT-PER-L ;
+  qudt:applicableUnit unit:PicoKAT-PER-L ;
+  qudt:applicableUnit unit:IU-PER-L ;
+  qudt:hasDimensionVector qkdv:A1E0L-3I0M0H0T-1D0 ;
+  rdfs:label "Catalytic Activity Concentration"@en ;
+  skos:broader <http://qudt.org/vocab/quantitykind/AmountOfSubstanceConcentrationOfB> ;
+.
+
+# This amends the existing unit to link to the QK
+unit:IU-PER-L
+  a qudt:DerivedUnit ;
+  qudt:hasQuantityKind quantitykind:CatalyticActivityConcentration .
+
+
+unit:KAT-PER-L
+  a qudt:DerivedUnit ;
+  a qudt:Unit ;
+  dcterms:description "A unit of catalytic activity used especially in the chemistry of enzymes. A catalyst is a substance that starts or speeds a chemical reaction. Enzymes are proteins that act as catalysts within the bodies of living plants and animals. A catalyst has an activity of one katal if it enables a reaction to proceed at the rate of one mole per second. "^^rdf:HTML ;
+  qudt:plainTextDescription "A unit of catalytic activity used especially in the chemistry of enzymes. A catalyst is a substance that starts or speeds a chemical reaction. Enzymes are proteins that act as catalysts within the bodies of living plants and animals. A catalyst has an activity of one katal if it enables a reaction to proceed at the rate of one mole per second. "^^rdf:HTML ;
+  qudt:applicableSystem sou:SI ;
+  qudt:conversionMultiplier 0.001 ;
+  qudt:dbpediaMatch "http://dbpedia.org/resource/Katal"^^xsd:anyURI ;
+  qudt:definedUnitOfSystem sou:SI ;
+  qudt:derivedCoherentUnitOfSystem sou:SI ;
+  qudt:hasQuantityKind quantitykind:CatalyticActivityConcentration ;
+  qudt:hasDimensionVector qkdv:A1E0L-3I0M0H0T-1D0 ;
+  qudt:symbol "kat/L" ;
+  qudt:ucumCode "kat/L"^^qudt:UCUMcs ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
+  rdfs:label "Katal Per Litre"@en ;
+  rdfs:label "Katal Per Litre"@en-us 
+.
+
+unit:MilliKAT-PER-L
+  a qudt:DerivedUnit ;
+  a qudt:Unit ;
+  dcterms:description "A unit of catalytic activity used especially in the chemistry of enzymes. A catalyst is a substance that starts or speeds a chemical reaction. Enzymes are proteins that act as catalysts within the bodies of living plants and animals. A catalyst has an activity of one katal if it enables a reaction to proceed at the rate of one mole per second. "^^rdf:HTML ;
+  qudt:plainTextDescription "A unit of catalytic activity used especially in the chemistry of enzymes. A catalyst is a substance that starts or speeds a chemical reaction. Enzymes are proteins that act as catalysts within the bodies of living plants and animals. A catalyst has an activity of one katal if it enables a reaction to proceed at the rate of one mole per second. "^^rdf:HTML ;
+  qudt:applicableSystem sou:SI ;
+  qudt:conversionMultiplier 1.0  ;
```

Comment:
1 MilliKAT-PER-L is 0.000001 KAT-PER-M3.  It's 1000 times smaller than a KAT-PER-L, not 1000 times bigger.

Reply:
Updated